test/tests/components/ws-manager/cpu_burst_test.go

```diff
@@ -0,0 +1,154 @@
+// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package wsmanager
+
+import (
+	"context"
+	"strings"
+	"testing"
+	"time"
+
+	"sigs.k8s.io/e2e-framework/pkg/envconf"
+	"sigs.k8s.io/e2e-framework/pkg/features"
+
+	"github.com/gitpod-io/gitpod/common-go/log"
+	csapi "github.com/gitpod-io/gitpod/content-service/api"
+	daemon "github.com/gitpod-io/gitpod/test/pkg/agent/daemon/api"
+	wsapi "github.com/gitpod-io/gitpod/test/pkg/agent/workspace/api"
+	"github.com/gitpod-io/gitpod/test/pkg/integration"
+	wsmanapi "github.com/gitpod-io/gitpod/ws-manager/api"
+	corev1 "k8s.io/api/core/v1"
+)
+
+const (
+	expectedMinCpu   = 200_000
+	expectedBurstCpu = 600_000
+)
+
+func TestCpuBurst(t *testing.T) {
+	f := features.New("cpulimiting").WithLabel("component", "ws-manager").Assess("check cpu limiting", func(_ context.Context, t *testing.T, cfg *envconf.Config) context.Context {
```

Comment:
I think we have to add a check if there is a configuration to enable CPU limit and if there isn't we should skip the test.